src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateHoldingEventHandler.java

```diff
@@ -54,8 +56,8 @@ public CreateHoldingEventHandler(Storage storage) {
   public CompletableFuture<DataImportEventPayload> handle(DataImportEventPayload dataImportEventPayload) {
     CompletableFuture<DataImportEventPayload> future = new CompletableFuture<>();
     try {
-      if (dataImportEventPayload.getContext() == null || dataImportEventPayload.getContext().isEmpty()
-        || !dataImportEventPayload.getContext().containsKey(MARC_BIBLIOGRAPHIC.value())) {
+      if (dataImportEventPayload.getContext().isEmpty()
```

Comment:
```suggestion
      if (dataImportEventPayload.getContext() == null
```
ref to context can be null

Reply:
Done.